test/libsolidity/SolidityParser.cpp

```diff
@@ -540,7 +540,7 @@ BOOST_AUTO_TEST_CASE(if_statement)
 {
 	char const* text = "contract test {\n"
 					   "  function fun(uint256 a) {\n"
-					   "    if (a >= 8) return 2; else { var b = 7; }\n"
+					   "    if (a >= 8) { return 2 }; else { var b = 7; }\n"
```

Comment:
Thanks, I fixed it.

I can revert these commits if you'd like. The only reason why I did it was because I was grepping through all the Solidity code for `else` and, in doing so, I thought that these test cases looked a little inconsistent so I just fixed them up.